pom.xml

```diff
@@ -85,10 +86,11 @@
     <module>orientdb</module>
     <module>redis</module>
     <module>voldemort</module>
-    <module>distribution</module>
     <!--<module>mapkeeper</module>-->
     <!--module>nosqldb</module-->
     <module>couchbase</module>
+    <module>tarantool</module>
+    <module>distribution</module>
```

Comment:
Yeah, that's a problem with how we're building our distribution. It'll be fixed properly in #250, so leaving this work-around here is fine.